moveit_kinematics/cached_ik_kinematics_plugin/include/moveit/cached_ik_kinematics_plugin/cached_ik_kinematics_plugin-inl.h

```diff
@@ -88,25 +87,7 @@ bool CachedMultiTipIKKinematicsPlugin<KinematicsPlugin>::initialize(const moveit
 }
 
 template <class KinematicsPlugin>
-bool CachedMultiTipIKKinematicsPlugin<KinematicsPlugin>::initialize(const std::string& robot_description,
-                                                                    const std::string& group_name,
-                                                                    const std::string& base_frame,
-                                                                    const std::vector<std::string>& tip_frames,
-                                                                    double search_discretization)
-{
-  // call initialize method of wrapped class
-  if (!KinematicsPlugin::initialize(robot_description, group_name, base_frame, tip_frames, search_discretization))
-    return false;
-
-  std::string cache_name = base_frame;
-  std::accumulate(tip_frames.begin(), tip_frames.end(), cache_name);
-  CachedIKKinematicsPlugin<KinematicsPlugin>::cache_.initializeCache(robot_description, group_name, cache_name,
-                                                                     KinematicsPlugin::getJointNames().size());
-  return true;
-}
```

Comment:
I couldn't get this initialize to compile successfully, it kept trying to match it to the RobotModel template. But I'm not sure why this KinematicsBase::initialize even exists as it doesn't take in a Node

Reply:
initialize() only exists because the plugin constructor of doesn't support an arbitrary number of parameters.

Reply:
oh, nevermind. I missed that you were only talking about the non-node version. Removing it is completely fine